frontend/src/router.js

```diff
@@ -35,56 +35,60 @@ import Member from './containers/member';
 import MemberInfo from './components/member/info';
 import MemberRaw from './components/member/raw';
 
-const AppRouter = () => {
-    return (
-       <Router history={hashHistory}>
-            <Route path="/" component={App}>
-                <IndexRedirect to="/cluster" />
-                <Route path="/cluster" component={Cluster} />
 
-                <Route path="/members" component={Members} />
-                <Route path="/members/:memberId" component={Member}>
-                    <IndexRedirect to="/members/:memberId/info" />
-                    <Route path="/members/:memberId/info" component={MemberInfo} />
-                    <Route path="/members/:memberId/raw" component={MemberRaw} />
-                </Route>
+class AppRouter extends Component {
```

Comment:
Why did you change it to a component? A stateless function (like it was) will do the job?